keras/src/ops/operation.py

```diff
@@ -119,6 +123,17 @@ def __new__(cls, *args, **kwargs):
         to manually implement `get_config()`.
         """
         instance = super(Operation, cls).__new__(cls)
+        if backend.backend() == "jax":
+            if is_nnx_backend_enabled():
+                try:
+                    from flax import nnx
+
+                    vars(instance)["_object__state"] = nnx.object.ObjectState()
```

Comment:
We've talked about this, I'm really confused why it needs to go here instead of the Layer class, all the more so as `Operation` doesn't extend `nnx.Module`, so NNX should not care about Operations that are not Layers.

Maybe it's just a question of placing it early enough in `Layer.__init__`.